### PR TITLE
drain: add timestamps to cli output

### PR DESCRIPTION
**Pull request by schmichael**
Also toggle eligibility output based on value

Output:

image

### CODE REVIEW

File: command/node_drain.go [hunk 1]
```diff
@@ -271,16 +271,20 @@ func (c *NodeDrainCommand) Run(args []string) int {
...
+		if enable {
+			c.Ui.Output(fmt.Sprintf("Node %q drain strategy set", node.ID))
+		} else {
+			c.Ui.Output(fmt.Sprintf("Node %q drain strategy unset", node.ID))
+		}
 	}
 
 	if enable && !detach {
+		c.Ui.Output(fmt.Sprintf("%s Node %q drain strategy set", formatTime(time.Now()), node.ID))
+		c.Ui.Output(" (Ctrl-C to stop monitoring: will not cancel the node drain)")
```
Comment (dadgar) [inline command/node_drain.go]: Any reason for the space in front? Also What about putting this as the first line?

File: command/node_drain.go [hunk 2]
```diff
@@ -271,16 +271,20 @@ func (c *NodeDrainCommand) Run(args []string) int {
...
 		outCh := client.Nodes().MonitorDrain(context.Background(), node.ID, meta.LastIndex, ignoreSystem)
 		for msg := range outCh {
-			c.Ui.Output(msg)
+			c.Ui.Output(fmt.Sprintf("%s %s", formatTime(time.Now()), msg))
```
Comment (dadgar) [inline command/node_drain.go]: Did we want colons?

Comment (schmichael) [reply command/node_drain.go]: Like?
```
2018-03-30T00:04:53Z: All allocations on
```

I don't think it really clarifies much.

File: command/node_drain.go [hunk 3]
```diff
@@ -271,16 +271,30 @@ func (c *NodeDrainCommand) Run(args []string) int {
...
+		if enable {
+			c.Ui.Output(fmt.Sprintf("Node %q drain strategy set", node.ID))
+		} else {
+			c.Ui.Output(fmt.Sprintf("Node %q drain strategy unset", node.ID))
+		}
 	}
 
 	if enable && !detach {
+		now := time.Now()
+		c.Ui.Warn(fmt.Sprintf("%s: Ctrl-C to stop monitoring: will not cancel the node drain", formatTime(now)))
```
Comment (dadgar) [inline command/node_drain.go]: Info

**Pull request merged**